pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -131,6 +131,13 @@ func (cgc *containerGC) removeOldestN(containers []containerGCInfo, toRemove int
 // removeSandbox removes the sandbox by sandboxID.
 func (cgc *containerGC) removeSandbox(sandboxID string) {
 	glog.V(4).Infof("Removing sandbox %q", sandboxID)
+	// In normal cases, kubelet should've already called StopPodSandbox before
+	// GC kicks in. To guard against the rare cases where this is not true, try
```

Comment:
I expanded the explanation, but didn't provide a concrete example on purpose. I think the concrete examples involves too much runtime implementation details (which is not universal), the inner workings of kubernetes (which can change anytime).